src/guides/v2.4/graphql/mutations/assign-customer-to-guest-cart.md

```diff
@@ -0,0 +1,96 @@
+---
+group: graphql
+title: assignCustomerToGuestCart mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `assignCustomerToGuestCart` mutation assigns a logged in customer and merge customer's shopping cart items to the specified guest shopping cart.
+
```

Comment:
I don't understand when this mutation would be used. Is it when a logged in user logs out or if the session expires?

Mention that a customer token is required (assuming that's true).

Add a new paragraph that describes the difference between this mutation and `mergeCarts`. Add a similar statement to the `mergeCarts` mutation.

Reply:
thanks for the note, I've added an additional text about the difference